languages/javascript/exercises/concept/array-destructuring/example.js

```diff
@@ -0,0 +1,30 @@
+export function getFirstCard(deck) {
+  const [first] = deck
+
+  return first
+}
+
+export function getSecondCard(deck) {
+  const [, second] = deck
+
+  return second
+}
+
+export function swapTopTwoCards(deck) {
+  ;[deck[0], deck[1]] = [deck[1], deck[0]]
+
+  return deck
+}
+
+export function discardTopCard(deck) {
+  const [first, ...rest] = deck
+
+  return [first, rest]
+}
+
+export function insertFaceCards(deck) {
+  const faceCards = ['jack', 'queen', 'king']
+  const [topCard, ...rest] = deck
+
+  return [topCard, ...faceCards, ...rest]
+}
```

Comment:
Should use the top-level constant (as mentioned)

```suggestion
export function insertFaceCards([head, ...tail]) {
  return [head, ...FACE_CARDS, ...tail]
}
```

Reply:
Done.